Pendo Feedback/README.md

```diff
@@ -0,0 +1,109 @@
+# Indexing Pendo Feeback (Formerly Receptive.io) Using the Generic REST API Connector
+
+## Use Case
+This example shows how to retrieve Features(cases) submit to the Pendo Feedback platform and the comments on each Feature.
+
+## Prerequisites
+To fully understand how to use this example, you must:
+1. Have a Coveo Platform organization.
+2. Learn about [Coveo Connectivity](https://docs.coveo.com/en/1702/).
+3. Learn [how to configure a Generic REST API source](https://docs.coveo.com/en/1896/).
+
+## Instructions
+1. [Get a Pendo Feeback API key](https://app.pendo.io/).
```

Comment:
Would there be a better link to explain how to generate the API key? I haven't found anything apart from their API doc https://developers.pendo.io/docs/?bash#options which briefly touch on the process to get an API in the Options section.

Reply:
This is the closest I could find: https://support.pendo.io/hc/en-us/articles/360042025452-Salesforce-Integration-Setup-for-Feedback#h_6cb2d977-ee2b-4b8f-aa24-66455c2d4b9d